server/container/guice/jpa-guice/pom.xml

```diff
@@ -194,7 +194,7 @@
                  <dependency>
                     <groupId>org.apache.derby</groupId>
                     <artifactId>derby</artifactId>
-                    <scope>test</scope>
+                     <scope>compile</scope>
```

Comment:
Yes, if user would like to use another database, so they need to change property file (james-database.properties) with the right driver, url ... and they need jar package for that driver.
In our case, we have to launch server and provision some initial data.
And we have driver class is: database.driverClassName=org.apache.derby.jdbc.EmbeddedDriver
So I think it need to modify that scope so that the jar file is included when we package server.
Is that correct?